crates/re_space_view_bar_chart/Cargo.toml

```diff
@@ -29,4 +29,3 @@ re_viewer_context.workspace = true
 
 egui_plot.workspace = true
 egui.workspace = true
-vec1.workspace = true
```

Comment:
any use of vec1 left? Might be able to remove it from the workspace Cargo.toml now (?)

Reply:
`crates/re_time_panel/src/time_axis.rs` still uses it